Core/Src/main.c

```diff
@@ -96,8 +104,33 @@ int main(void)
   while (1)
   {
     /* USER CODE END WHILE */
+	  /* USER CODE BEGIN 3 */
+	  uint8_t transmit_data[]={0x01,0x80,0x00};
```

Comment:
Since the size of this array is known, why not give it a size in initialization? Also, you could add a `const` keyword to it which can make the transmit_data read-only.